bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -113,6 +113,16 @@
 		<state pattern="%.1f %unit%" step="0.5"/>
 	</channel-type>
 
+	<channel-type id="targetTemperature" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<description>Target temperature (read only)</description>
+		<tags>
+			<tag>Temperature</tag>
+		</tags>
```

Comment:
Something missing. Is it a measurement, a setpoint ??

Reply:
in fact it's neither. See previus answser. If one has to be chosen then setpoint is better.

Reply:
In that case, please set no tag at all

Reply:
done